src/styles/blocks/main.css

```diff
@@ -0,0 +1,81 @@
+.product-card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.product-card__image {
+  width: 100%;
+  padding: 0 3px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  margin-top: 37px;
+  line-height: var(--title-line-height);
+  font-weight: 500;
+  color: rgba(6, 11, 53, 1);
+}
+
+.product-card__subtitle {
+  margin-top: 4px;
+  color: rgba(97, 96, 112, 1);
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.product-card__rates {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  line-height: 16px;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: rgb(6, 11, 53);
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin-top: 24px;
+}
+
+.product-card__price-title {
+  font-size: 12px;
+  line-height: var(--title-line-height);
+  font-weight: 400;
+  color: #616070;
+}
+
+.product-card__price-block {
+  line-height: var(--title-line-height);
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__buy-link {
+  display: block;
+  text-align: center;
+  padding: 11px 0;
+  margin-top: 16px;
+  border: 1px solid transparent;
+  background-color: var(--color-light-blue);
```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)